### PR TITLE
[Dy2Stat]Support to transform sequence assignments and multi-target assignments

### PR DESCRIPTION
**Pull request by liym27**
As the title, support to transform sequence assignments and multi-target assignments to normal assignments.

Code transformation is as follows:
- Before:
```Python
a, b = c, d
```
- After:
```
a = c
b = d
```

- Before:
```Python
a, b = c, d = x
```
- After:
```
 c,d = x
 a = c
 b = d
```

### CODE REVIEW

File: python/paddle/fluid/dygraph/dygraph_to_static/utils.py [hunk 1]
```diff
@@ -927,3 +927,83 @@ def _get_enum_idx_name(self):
...
+        if len(targets) != len(values):
+            return node
+
+        for target, value in zip(targets, values):
+            assign_node = gast.Assign(targets=[target], value=value)
+            new_nodes.append(assign_node)
+
+        return new_nodes
+
+    def _parse_mul_target_assign(self, node):
```
Comment (chenwhql) [inline python/paddle/fluid/dygraph/dygraph_to_static/utils.py]: 函数名要不要改成`_parse_multi_target_assign`，mul容易误解为乘法

**Review by chenwhql — COMMENTED** [review]
LGTM

File: python/paddle/fluid/dygraph/dygraph_to_static/utils.py (hunk 1 above)
Comment (liym27) [reply python/paddle/fluid/dygraph/dygraph_to_static/utils.py]: 好的，感谢建议！已修改

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Review by zhhsplendid — APPROVED** [review]

**Pull request merged**